src/main/java/com/alex/eshop/webconfig/WebSecurityConfig.java

```diff
@@ -20,14 +19,12 @@
 import java.util.stream.Collectors;
 
 @Configuration
-@EnableGlobalMethodSecurity(prePostEnabled = true)
-public class WebSecurityConfiguration extends WebSecurityConfigurerAdapter {
-
-    @Override
+@EnableMethodSecurity
+public class WebSecurityConfig {
     protected void configure(HttpSecurity http) throws Exception {
```

Comment:
function `void configure(HttpSecurity http)` is not used now. So `requestMatchers()` does not work anymore. This is a bug